### ISSUE TITLE
[BUG] IM6001-BTP01 (smartthings) button

### ISSUE THREAD

**Issue by deonte1threatt** [xaviml/controllerx#397]
# Bug report

## Description

Smartthings button does not fire "single_click" and "double_click" instead firing "single" and "double"

* Devices involved:
IM6001-BTP01 (smartthings) button
* Integration: z2m
* AppDaemon version: 0.7.1 
* ControllerX version: 4.16.0
* HACS version (if installed from there): 1.18.0
* Home Assistant Core version: 2021.12.6

## AppDaemon app configuration

bathroom_button:
  module: controllerx
  class: SmartThingsButtonLightController
  controller: sensor.main_bathroom_button_action
  integration: z2m
  light: light.main_bathroom

## Logs
MQTT Showing:
Message 641 received on zigbee2mqtt/Main Bathroom Button/action at 5:35 PM:
single
QoS: 0 - Retain: false
```text
(optional) Add relevant AppDaemon / Home Assistant logs which could help tackle the problem.
```

## Additional Context

Add any other context or screenshots about the bug here.

**Comment by xaviml** [comment]
Hi @deonte1threatt,

Sorry to hear this. I will change this for the next release. In the meantime, you can do:

```yaml
bathroom_button:
  module: controllerx
  class: SmartThingsButtonLightController
  controller: sensor.main_bathroom_button_action
  integration: z2m
  light: light.main_bathroom
  mapping:
    single: toggle
    double: on_full_brightness
```

This will overwrite the default mapping and you can specify which event to listen to.

Regards,
Xavi M.

**Comment by xaviml** [comment]
Hi @deonte1threatt,

I have fixed this in [ControllerX v4.18.0](https://github.com/xaviml/controllerx/releases/tag/v4.18.0).

Regards,
Xavi M.
